docs/source/getting-started/intro.rst

```diff
@@ -0,0 +1,312 @@
+A Brief Introduction
+====================
+
+In this section, we provide a brief overview of the major functionality
+of nbodykit, as well as an introduction to some of the technical jargon
+needed to get up and running quickly. We try to familiarize the user with the
+various aspects of nbodykit needed to take full advantage of nbodykit's
+computing power. This section also serves as a nice outline of the documentation,
+with links to more detailed descriptions included throughout.
+
+.. contents::
+   :depth: 2
+   :local:
+   :backlinks: none
+
+The ``lab`` framework
+---------------------
+
+A core design goal of nbodykit is maintaining an interactive user
+experience, allowing the user to quickly experiment and play around
+with data sets and statistics, while still leveraging the power of
+parallel processing when necessary. Motivated by the power of
+`Jupyter notebooks`_, we adopt a "lab" framework for nbodykit, where all of the
+necessary data containers and algorithms can be imported from a single module:
+
+.. code-block:: python
+
+  from nbodykit.lab import *
+
+  [insert cool science here]
+
+See the documentation for :mod:`nbodykit.lab` for a full list of the
+imported members in this module.
+
+With all of the necessary tools now in hand, the user can easily load
+a data set, compute statistics of that data via one of the
+built-in algorithms, and save the results in just a few lines. The end
+result is a reproducible scientific result, generated from clear
+and concise code that flows from step to step.
+
+Parallel computation with MPI
+-----------------------------
```

Comment:
@rainwoodman let me know if you have any suggestions for this section in particular